### PR TITLE
1 products api creation

### LINKED ISSUE #1: products-api-creation

### PR DESCRIPTION
**Pull request by SruthiKonda12**

### CODE REVIEW

File: config/email_credentials.py
Comment (QodeFarm) [inline config/email_credentials.py:1]: Do we need this file? I don't think so

File: config/settings.py [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+    "TOKEN_OBTAIN_SERIALIZER": "rest_framework_simplejwt.serializers.TokenObtainPairSerializer",
+    "TOKEN_REFRESH_SERIALIZER": "rest_framework_simplejwt.serializers.TokenRefreshSerializer",
+    "TOKEN_VERIFY_SERIALIZER": "rest_framework_simplejwt.serializers.TokenVerifySerializer",
+    "TOKEN_BLACKLIST_SERIALIZER": "rest_framework_simplejwt.serializers.TokenBlacklistSerializer",
+    "SLIDING_TOKEN_OBTAIN_SERIALIZER": "rest_framework_simplejwt.serializers.TokenObtainSlidingSerializer",
+    "SLIDING_TOKEN_REFRESH_SERIALIZER": "rest_framework_simplejwt.serializers.TokenRefreshSlidingSerializer",
+}
+
+#DJOSER SETTINGS
+DJOSER = {
```
Comment (QodeFarm) [inline config/settings.py]: delete this DJOSER section

File: config/settings.py [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+
+
+REST_FRAMEWORK = {    
+    #'DEFAULT_PERMISSION_CLASSES': ('rest_framework.permissions.IsAuthenticated',),  #For Securing API Endpoint
+    'DEFAULT_AUTHENTICATION_CLASSES': ('rest_framework_simplejwt.authentication.JWTAuthentication',
+                                       'rest_framework.authentication.TokenAuthentication',),
+    'DEFAULT_VERSION': 'v1',
+}
+
+SIMPLE_JWT = {
```
Comment (QodeFarm) [inline config/settings.py]: Delete this SIMPLE_JWT section

File: config/urls.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+from config import settings
+from config.views import api_links
+from rest_framework import permissions
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+# from apps.users.custom_schema_generator import CustomOpenAPISchemaGenerator  
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="NTHRAS",
```
Comment (QodeFarm) [inline config/urls.py]: Change this RUDHRA

File: database/rudhra.sql [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+   - Where possible, offload data processing and business logic to the application level to leverage application caching and reduce database load.
+   - Regularly review and optimize SQL queries used by the application, especially those that are executed frequently or involve large datasets.
+
+10. Database Maintenance:
+    - Regularly perform database maintenance tasks such as analyzing tables, optimizing indexes, and cleaning up unused data or tables to ensure optimal performance.
+    - Plan for regular backups and establish a robust disaster recovery plan to safeguard your data.
+
+By following these best practices, developers can ensure that the database layer of the Nthras Product ERP system remains robust, performant, and scalable to support the evolving needs of the business.
+
+*/
```
Comment (QodeFarm) [inline database/rudhra.sql]: Remove above comments

File: config/email_credentials.py
Comment (SruthiKonda12) [reply config/email_credentials.py:1]: no need Thulasi

**Pull request merged**